### PR TITLE
[vcpkg baseline][monkeys-audio] Update to 5.59

### PR DESCRIPTION
**Pull request by NancyLi1013**
**Describe the pull request**

- What does your PR fix? Fixes #

Released MAC_SDK files are currently only provided for download for a
short period of time and replaced frequently by newer versions.

```
Building package monkeys-audio[core]:x64-windows...
-- Downloading https://monkeysaudio.com/files/MAC_SDK_548.zip...
-- Downloading https://monkeysaudio.com/files/MAC_SDK_548.zip... Failed. Status: 22;"HTTP response code said error"
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:182 (message):
      
      Failed to download file.
      If you use a proxy, please set the HTTPS_PROXY and HTTP_PROXY environment
      variables to "https://user:***@your-proxy-ip-address:port/".
      Otherwise, please submit an issue at https://github.com/Microsoft/vcpkg/issues

```
So update the current version in vcpkg to the latest version 5.59.

Related PR #7733

Note: feature has been passed with the following triplets:

-  x86-windows
-  x64-windows

### CODE REVIEW

File: ports/monkeys-audio/portfile.cmake [hunk 1]
```diff
@@ -2,19 +2,21 @@ vcpkg_fail_port_install(ON_TARGET "UWP" "OSX" "Linux")
...
 )
 
 vcpkg_extract_source_archive_ex(
     OUT_SOURCE_PATH SOURCE_PATH
     ARCHIVE ${ARCHIVE}
     NO_REMOVE_ONE_LEVEL
-    PATCHES fix-project-config.patch
+    PATCHES 
+	    fix-project-config.patch
+	    remove-cereficate-step.patch
```
Comment (NancyLi1013) [inline ports/monkeys-audio/portfile.cmake:19]:
```suggestion
    PATCHES 
        fix-project-config.patch
        remove-cereficate-step.patch
```

File: ports/monkeys-audio/portfile.cmake [hunk 2]
```diff
@@ -2,19 +2,21 @@ vcpkg_fail_port_install(ON_TARGET "UWP" "OSX" "Linux")
...
 )
 
 vcpkg_extract_source_archive_ex(
     OUT_SOURCE_PATH SOURCE_PATH
     ARCHIVE ${ARCHIVE}
     NO_REMOVE_ONE_LEVEL
-    PATCHES fix-project-config.patch
+    PATCHES 
+        fix-project-config.patch
+        remove-cereficate-step.patch
```
Comment (c72578) [inline ports/monkeys-audio/portfile.cmake:19]: There is a typo in the filename of the patch:
cereficate->certificate

Comment (NancyLi1013) [reply ports/monkeys-audio/portfile.cmake:19]: It's my mistake. Thanks for your kindly reminder.

**Review by JackBoosY — APPROVED** [review]

**Comment by JackBoosY** [comment]
Ping @strega-nil for merge this PR first.

File: ports/monkeys-audio/portfile.cmake [hunk 3]
```diff
@@ -2,19 +2,21 @@ vcpkg_fail_port_install(ON_TARGET "UWP" "OSX" "Linux")
...
 )
 
 vcpkg_extract_source_archive_ex(
     OUT_SOURCE_PATH SOURCE_PATH
     ARCHIVE ${ARCHIVE}
     NO_REMOVE_ONE_LEVEL
-    PATCHES fix-project-config.patch
+    PATCHES 
+	    fix-project-config.patch
+	    remove-ceretificate-step.patch
```
Comment (c72578) [inline ports/monkeys-audio/portfile.cmake:19]: The typo in the filename is still not completely fixed:
ceretificate->certificate

Comment (NancyLi1013) [reply ports/monkeys-audio/portfile.cmake]: Sorry again for my fault.

**Comment by strega-nil** [comment]
Thanks Nancy!

**Pull request merged**